app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ApplicationRecord
   has_many   :miq_widget_contents, :dependent => :destroy
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_reports, :dependent => :nullify
+  has_many   :service_orders, :dependent => :nullify
```

Comment:
I can see wanting to run reports on order history either by tenant or globally. When we add costing, we will for sure not want to lose any order history and would certainly be nice to know who placed the orders, even if the user records have been removed.